python/graphstorm/trainer/ep_trainer.py

```diff
@@ -112,6 +112,9 @@ def fit(self, train_loader, n_epochs,
         for epoch in range(n_epochs):
             model.train()
             t0 = time.time()
+            if freeze_input_layer_epochs <= epoch:
+                self._model.unfreeze_input_encoder()
+            # TODO(xiangsx) Support unfreezing gnn encoder and decoder
```

Comment:
we need to have a regression test to verify the BERT+GNN model training.

Reply:
We do.